changelog.json

```
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "3.1.0": [
+      "[Added] You can diff a consecutive range of commits - #14682",
+      "[Added] Improve submodule support with a new diff screen with more detailed information - #15176",
+      "[Fixed] Enable update from upstream default branch when contributing to a fork - #15299",
+      "[Fixed] Fix crash installing updates that were downloaded a long time ago on macOS - #15275",
+      "[Fixed] Tooltips are positioned accurately when the user prefers reduced motion - #15249",
+      "[Fixed] Add option to disable Undo Commit popup - #15134. Thanks @geophilusd!",
```

```suggestion
      "[Improved] Add option to disable Undo Commit confirmation - #15134. Thanks @geophilusd!",
```
Subjective nit -> This feels like an improvement as I don't think it was bug that you could not undo the commit popup.